### ISSUE TITLE
[Accessibility, a11y] Color contrast ratio of tree view `description` text is 4.381:1 which is less than 4.5:1

### ISSUE THREAD

**Issue by MicroFish91** [microsoft/vscode#237130]
- VS Code Version: 1.97.0-insider
- OS: Windows
- Color Theme: Dark+ (Default Dark+)

This issue is being escalated from here: https://github.com/microsoft/vscode-azurefunctions/issues/4332

## Steps to Reproduce:

1.  Install the Azure Resources extension `ms-azuretools.vscode-azureresourcegroups`
2. Click on the `Azure` tab
3. Observe the "`Remote`" description in the `Resources` tree view

![Image](https://github.com/user-attachments/assets/1d367b22-087d-4448-ba88-694c984ce751)

## Actual:
Color contrast ratio of tree view's `description` text is 4.381:1 which is less than 4.5:1

## Expected:
Color contrast ratio of tree view's `description` text should be greater than or equal to 4.5:1

This should reproduce for any tree view description in default dark mode.

**Comment by MicroFish91** [comment]
Actually maybe hold off on this, I tested recently and am getting a higher contrast ratio than the one that was listed in the linked issue that was referenced.

**Comment by MicroFish91** [comment]
Testers confirming it still reproduces.  They are using a Windows contrast analyzer, I was using a web tool on macOS.  I haven't been able to reproduce on macOS, but it seems multiple testers have reproduced using the same Windows tool shown in the screenshot.

**Comment by PoojaNamde** [comment]
@MicroFish91 we have logged https://github.com/microsoft/vscode/issues/246455 new bug to track this issue. Could you please close this.
